specs/protocol/configurability.md

```diff
@@ -0,0 +1,78 @@
+- **Service Roles**: These roles are used to manage the day-to-day
+  operations of the chain, and therefore are often hot wallets.
+
+## Consensus Parameters
+
+| Config Property                       | Description                                                                                                                  |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| Batch Inbox address                   | L1 address where calldata/blobs are posted (see [Batcher Transaction](../glossary.md#batcher-transaction)).                         |
+| Chain ID                              | Unique ID of Chain used for TX signature validation.                                                                         |
+| Challenge Period                      | Length of time for which an output root can be removed, and for which it is not considered finalized.                                                                                                                                                             |
+| Data Availability                     | Where L2 transaction calldata is posted.                                                                                     |
+| Fault Proof Window / Challenge Window | Duration network participants have to challenge the integrity of an output root.                                             |
+| Fee margin                            | Markup on transactions compared to the raw L1 data cost.                                                                     |
+| Proxy Fee vault contracts                   | Proxy contracts which point to implementation contracts that dictate how user fees are distributed.                                                                       |
+| Gas limit                             | L2 block gas limit.                                                                                                          |
+| Gas Token                             | Native token used to pay for gas.                                                                                            |
+| Genesis state                         | Initial state at chain genesis, including code and storage of predeploys (all L2 smart contracts). See [Predeploy](../glossary.md#l2-genesis-block).                                            |
+| L1 smart contracts                    | The chain’s L1 smart contracts.                                                                                              |
+| L2 block time                         | Frequency with which blocks are produced as a result of derivation.                                                          |
+| Output frequency                      | Frequency with which output roots are submitted to L1.                                                                       |
```

Comment:
The output frequency is policy. Making it smaller means less time after doing a withdrawal on L2 until you can prove your withdrawal on L1